pkg/apiserver/slowquery/statement_gen.go

```diff
@@ -25,9 +27,15 @@ func genSelectStmt(tableColumns []string, reqJSONColumns []string) (string, erro
 
 	// We have both TiDB 4.x and TiDB 5.x columns listed in the model. Filter out columns that do not exist in current version TiDB schema.
 	fields = lo.Filter(fields, func(f Field, _ int) bool {
-		hasProjection := f.Projection != ""
```

Comment:
Sorry for my late reply.. Why is `hasProjection` no longer needed?

Reply:
I use `related` to replace it, the same as in the statement.